From 7244eac44be929fabd6ed1333f96929ef8da564f Mon Sep 17 00:00:00 2001
From: Jonathan Rajotte <jonathan.rajotte-julien@efficios.com>
Date: Tue, 19 Mar 2019 17:56:49 +0000
Subject: [PATCH] fix: tests: link libpause_consumer on liblttng-ctl

This preload test library uses symbols from liblttng-ctl which are
resolved when preloaded by GLIBC but not by MUSL.

Upstream-Status: Accepted [f667fbd7f8b9512f9943edb2597c226fcc424ee9]
Backported to 2.11 and 2.10.

Signed-off-by: Michael Jeanson <mjeanson@efficios.com>
---
 tests/regression/tools/notification/Makefile.am | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/tests/regression/tools/notification/Makefile.am b/tests/regression/tools/notification/Makefile.am
index 41adc69..a352bb8 100644
--- a/tests/regression/tools/notification/Makefile.am
+++ b/tests/regression/tools/notification/Makefile.am
@@ -20,7 +20,10 @@ FORCE_SHARED_LIB_OPTIONS = -module -shared -avoid-version \
 			   -rpath $(abs_builddir)
 
 libpause_consumer_la_SOURCES = consumer_testpoints.c
-libpause_consumer_la_LIBADD = $(top_builddir)/src/common/libcommon.la $(DL_LIBS)
+libpause_consumer_la_LIBADD = \
+     $(top_builddir)/src/common/libcommon.la \
+     $(top_builddir)/src/lib/lttng-ctl/liblttng-ctl.la \
+     $(DL_LIBS)
 libpause_consumer_la_LDFLAGS = $(FORCE_SHARED_LIB_OPTIONS)
 noinst_LTLIBRARIES = libpause_consumer.la
 
-- 
2.17.1

